styles/Makefile 2

```diff
@@ -0,0 +1,6 @@
+test: tests
```

Comment:
Why does it show this as changed?

Reply:
no idea.  I think it' s a duplicate file and can be ignored.

Reply:
I don't want to merge in a bunch of junk.
Pull requests should ideally be small and contain nothing but the changes that are needed.